src/ctia/http/exceptions.clj

```diff
@@ -34,10 +34,21 @@
   (logging/log! :error e (ex-message e))
   (ex/response-validation-handler e data request))
 
+(defn es-ex-data
+  [e data request]
+  (into
+   {:message (.getMessage e)
+    :stacktrace (->> (.getStackTrace e)
+                     (mapv str))
+    :request request
+    :data data}
+   (when (instance? ExceptionInfo e)
+     {:ex-data (ex-data e)})))
```

Comment:
I have finally solved the bug, but shouldn't we let this log enrichment?

Reply:
The message and the stacktrace are already logged when you put the exception as the 1st argument.  I would only add the `request`, `data` and `ex-data`

```Clojure
(defn es-ex-data
  [e data request]
  (let [exception-data (ex-data e)]
    (cond-> {:request request
             :data data}
      exception-data
      (assoc :ex-data exception-data))))
```